### ISSUE TITLE
[BUG] No Illinois App Splash Screen Displaying 

### ISSUE THREAD

**Issue by vburgett** [rokwire/illinois-app#3983]
In 5.1.29 on prod on an Android (pixel 6, version 14), we no longer see the Illinois app splash screen with the Alma Mater statue. 

https://github.com/rokwire/illinois-app/assets/1044905/4384aea9-337b-4f49-a9f5-1008a112d9d6

**Comment by dobromirdobrev** [comment]
@vburgett ,

The custom fullscreen app splash screen is not available anymore. Android had changed the appearance of the splash screens since API level 31 (e.g. Android 12) (ref: [Android documentation](https://developer.android.com/develop/ui/views/launch/splash-screen)). 

"Simply said", the splash screen from Android 12 onwards represents a small app icon, background color and an animation. So, we are limited to that behavior. 

In other words, we cannot bring back the Alma Mater statue image on full screen.

**Comment by vburgett** [comment]
With the majority of the Illinois app users being on iOS, I'm going to close this ticket. I am not sure a different Android splash screen is an issue or at least not a high priority one.
